[2025-09-11T03:31:00] ==== Debut sync_all ==== 
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2018-10-26 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2022-06-25 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2023-03-01 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2023-09-25 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2024-10-15 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2020-11-16 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2024-10-18 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2022-05-13 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2016-11-02 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2018-09-17 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2023-01-26 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2022-02-06 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2020-09-26 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2023-02-22 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2018-02-10 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2022-09-04 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2022-12-07 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2018-01-06 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2020-04-24 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2020-07-18 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2021-09-03 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2020-06-27 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2023-08-26 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2016-08-22 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2018-04-27 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2019-02-02 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2021-04-08 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2018-03-02 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2018-01-14 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2020-11-26 00:00:00) while time zone support is active.
  warnings.warn(
/home/andysags/.local/lib/python3.13/site-packages/django/db/models/fields/__init__.py:1671: RuntimeWarning: DateTimeField Startup.created_at received a naive datetime (2021-09-07 00:00:00) while time zone support is active.
  warnings.warn(
Startups locales absentes de la source distante: [33, 34, 35]
Traceback (most recent call last):
  File "/home/andysags/G-SVR-500-COT-5-1-survivor-10/env/incubator/manage.py", line 22, in <module>
    main()
    ~~~~^^
  File "/home/andysags/G-SVR-500-COT-5-1-survivor-10/env/incubator/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
    ~~~~~~~~~~~~~~~^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/core/management/base.py", line 416, in run_from_argv
    self.execute(*args, **cmd_options)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/core/management/base.py", line 460, in execute
    output = self.handle(*args, **options)
  File "/home/andysags/G-SVR-500-COT-5-1-survivor-10/env/incubator/import_api/management/commands/sync_all.py", line 12, in handle
    results = services.sync_all()
  File "/home/andysags/G-SVR-500-COT-5-1-survivor-10/env/incubator/import_api/services.py", line 603, in sync_all
    )
    
  File "/home/andysags/G-SVR-500-COT-5-1-survivor-10/env/incubator/import_api/services.py", line 513, in sync_news
    }
    
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/query.py", line 988, in update_or_create
    obj, created = self.select_for_update().get_or_create(
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        create_defaults, **kwargs
        ^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/query.py", line 946, in get_or_create
    return self.get(**kwargs), False
           ~~~~~~~~^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/query.py", line 629, in get
    num = len(clone)
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/query.py", line 366, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/models/sql/compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/andysags/.local/lib/python3.13/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/psycopg/cursor.py", line 97, in execute
    raise ex.with_traceback(None)
KeyboardInterrupt
